package.json

```diff
@@ -24,6 +24,7 @@
     "react-css-themr": "^2.1.2",
     "react-dom": "^15.6.1",
     "react-motion": "^0.5.0",
+    "react-player": "^0.20.0",
```

Comment:
Please remember to install the exact version so we'll avoid problems with package-lock.json later.